include/seqan/basic/alphabet_residue.h

```diff
@@ -482,8 +482,8 @@ unknownValueImpl(Rna5 *)
  *
  * @signature typedef SimpleType<unsigned char, Iupac_> Iupac;
  *
- * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 19 in this order: 'U'=0, 'T', 'A',
- * 'W', 'C', 'Y', 'M', 'H', 'G', 'K', 'R', 'D', 'S', 'B', 'V', 'N'=15.
+ * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 15 in this order: '='=0, 'A'=1, 'C',
```

Comment:
Im not so sure about the `=`/`U` thing. 0 encodes both `0` and `=`, as it is also document in the inverse tables above.
So I would rather put `U` than `=` since `U` is an actual Iupac character. Preferably, I would just put both?

Reply:
ah true. then maybe both